README.CentOS6.md

```diff
@@ -56,29 +114,50 @@ To perform this step manually, copy the sample file to the apache configuration
 cp docs/config/apache-site /etc/httpd/conf.d/apache-site.conf
 ```
 
-Ensure that paths and settings in this new file are populated appropriately for your server, e.g. replacing placeholders such as `%LORISROOT%`, `%PROJECTNAME%`, `%LOGDIRECTORY%`. Ensure that the DocumentRoot is pointing to the htdocs/ directory under your LORIS root (usually `/var/www/loris/htdocs`).
+Ensure that paths and settings in this new file are populated appropriately for your server, e.g. replacing placeholders such as `%LORISROOT% --> /var/www/loris`, `%PROJECTNAME% --> loris`, `%LOGDIRECTORY% --> /var/log/httpd/loris-error.log`. Ensure that the DocumentRoot is pointing to `/var/www/loris/htdocs`.
 
 Also ensure that your new `smarty/templates_c/` directory under the LORIS
 root is writable by Apache.
+```
+chmod 775 /var/www/loris/smarty/templates_c
+```
+Finally, restart apache:
+```
+sudo service httpd restart
+sudo service httpd status
+```
+# Install LORIS
+For the purpose of following LORIS conventions and easy understanding of all LORIS documentation, we recommend the following account names: 
+
+* lorisadmin : Linux user with sudo permission who will setup and manage Loris
+* lorisuser : MySQL user with limited (insert, delete, update, select...) permissions on the Loris database, for database transactions requested by the Loris front end
+* admin : default username for Loris front-end admin account (browser login)
+```
+cd /var/www/loris/tools
+./install.sh
+```
+point your URL to: `http://%IPADDRESS%/installdb.php`
 
-## SELinux
+(%IPADDRESS% will likely be the IP address of the VM you are ssh'ed into)
 
-Finally, SELinux will block MySQL connections from PHP. For now, we'll 
-disable SELinux (though on a production server you should learn how to
-properly configure it instead.)
+The web page will prompt you for the following information: 
 
-```bash
-echo 0 >/selinux/enforce
-```
+ * `Server Hostname`: localhost if your database if hosted on your VM or the IP address of your database server
 
-And restart apache
-```bash
-service httpd stop
-service httpd start
-```
+ * `Admin Username`: should be "root" 
+
+ * `Admin Password`: whatever password you set when running the `mysql_secure_installation` step
+
+ * `Database Name`: already set to "LORIS", can be customised
```

Comment:
I find this sentence confusing and I'm not sure what it's trying to get at. Is it already set or does it ask you to customize it?